charts/hocs-case-creator/values.yaml

```
@@ -26,3 +23,9 @@ hocs-generic-service:
       caseworkService: https://hocs-casework.{{ .Release.Namespace }}.svc.cluster.local
       workflowService: https://hocs-workflow.{{ .Release.Namespace }}.svc.cluster.local
       ignoreMessages: 'false'
+
+jobs:
+  process:
+    enabled: false
+    maxMessages: 100
+    schedule: '0 7-22 * * 1-5'
```

This is a arbitrary schedule till one is fully decided. Likely should be this for now:

```suggestion
    schedule: '0 8-18 * * 1-5'
```